saos-webapp/src/main/webapp/WEB-INF/view/judgment/judgmentDetails.jsp

```diff
@@ -3,7 +3,9 @@
 
 <script>
 $(document).ready(function() {
-	jsInitInJudgmentDetails();
+	jsInitInJudgmentDetails({
+		pageTitle: `<saos:caseNumber items='${judgment.caseNumbers}'/>` 
+	});
```

Comment:
And now to know what's going on in here I have to find jsInitInJudgmentDetails used only in this place.